app/src/main/kotlin/com/wire/android/ui/userprofile/image/AvatarPickerViewModel.kt

```diff
@@ -72,6 +79,15 @@ class AvatarPickerViewModel @Inject constructor(
         }
     }
 
+    fun postProcessAvatarImage(imgUri: Uri) {
+        viewModelScope.launch {
+            withContext(Dispatchers.IO) {
+                postProcessCapturedAvatar(imgUri, context)
```

Comment:
I think we can move functions like this to an object for example AvatarImageProcessor or AvatarImageManager and inject it into the viewmodel, that way we can also mock the object when unit testing the view model, but I already asked lots of changes in this PR so feel free to do it later if you feel like :P

Reply:
Done ! :)